src/services/productService.ts

```
@@ -39,4 +45,64 @@ export class ProductService {
+    if (totalElements > 1000) {
+      console.warn('대량 데이터 조회 시 성능 저하 가능');
+    }
+
+    const response = await api.get<SellerProductListApiResponse>('/products', {
+      params: {
+        page: 0,
+        size: totalElements,
+      },
+    });
+
+    if (!response.data.success) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)